src/vellum/workflows/inputs/dataset_row.py

```diff
@@ -18,14 +18,16 @@ class DatasetRow(UniversalBaseModel):
         id: Optional unique identifier for the dataset row
         label: String label for the dataset row
         inputs: BaseInputs instance or dict containing the input data
-        workflow_trigger_id: Optional Trigger identifying the workflow trigger class for this scenario
+        workflow_trigger: Optional Trigger instance for this scenario (can be passed as 'trigger' kwarg)
```

Comment:
[blocking] there's a mismatch here between the codegen (`trigger`) and the datamodel (`workflow_trigger`). I'd just pick the former